.github/workflows/app_build.yml

```diff
@@ -190,33 +199,65 @@ jobs:
             rm /usr/local/lib/pkgconfig/libcrypto.pc
             rm /usr/local/lib/pkgconfig/libssl.pc
             rm /usr/local/lib/pkgconfig/openssl.pc
-
-            echo "Installing IFW"
-            curl -L -O https://download.qt.io/official_releases/qt-installer-framework/4.6.1/QtInstallerFramework-macOS-x64-4.6.1.dmg
-            hdiutil attach -mountpoint ./qtfiw_installer QtInstallerFramework-macOS-x64-4.6.1.dmg
-            echo "ls ./qtfiw_installer"
-            ls ./qtfiw_installer
-            echo "ls ./qtfiw_installer/QtInstallerFramework-macOS-x64-4.6.1.app/"
-            ls ./qtfiw_installer/QtInstallerFramework-macOS-x64-4.6.1.app/
-            echo "ls ./qtfiw_installer/QtInstallerFramework-macOS-x64-4.6.1.app/Contents/"
-            ls ./qtfiw_installer/QtInstallerFramework-macOS-x64-4.6.1.app/Contents/
-            echo "ls ./qtfiw_installer/QtInstallerFramework-macOS-x64-4.6.1.app/Contents/MacOS"
-            ls ./qtfiw_installer/QtInstallerFramework-macOS-x64-4.6.1.app/Contents/MacOS
-            echo "ls ./qtfiw_installer/QtInstallerFramework-macOS-x64-4.6.1.app/Contents/Resources"
-            ls ./qtfiw_installer/QtInstallerFramework-macOS-x64-4.6.1.app/Contents/Resources
-
-            sudo ./qtfiw_installer/QtInstallerFramework-macOS-x64-4.6.1.app/Contents/MacOS/QtInstallerFramework-macOS-x64-4.6.1 --verbose --script ./ci/install_script_qtifw.qs
-            ls ~
-            ls ~/Qt/ || true
-            ls ~/Qt/QtIFW-4.6.1 || true
-            echo "~/Qt/QtIFW-4.6.1/bin/" >> $GITHUB_PATH
           fi;
         fi;
 
         cmake --version
         ccache --show-config || true
         ccache --zero-stats || true
 
+    - name: "Configure for codesigning"
+      if: runner.os == 'macOS'
+      run: |
+        set -x
+        cd $RUNNER_TEMP
+        mkdir codesigning && cd codesigning
+        # ----- Create certificate files from secrets base64 -----
+        echo "${{ secrets.MACOS_DEVELOPER_ID_APPLICATION_CERTIFICATE_P12_BASE64 }}" | base64 --decode > certificate_application.p12
```

Comment:
Is this certificate going to expire every year?  Maybe we can set up a reminder to renew it two weeks before it expires?

Reply:
I'll check tomorrow but I think only in 2026 or 2027